scripts/us_census/geojsons_low_res/generate_mcf.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Generates the file low_res_geojsons.mcf with the desired simplified geojsons
+for US states.
+
+    Typical usage:
+    python3 generate_mcf.py
+"""
+import geopandas as gpd
+import download
+import simplify
+import plotter
+import geojson
+import copy
+import tempfile
+import matplotlib.pyplot as plt
+
+
+class McfGenerator:
+    PLACE_TO_IMPORT = 'country/USA'
+
+    def __init__(self):
+        self.downloader = download.GeojsonDownloader()
+        self.simplifier = simplify.GeojsonSimplifier()
+        self.simple_geojsons = {}
+
+    def generate_simple_geojsons(self, verbose=True):
+        self.downloader.download_data(place=self.PLACE_TO_IMPORT)
+        for geoid, coords in self.downloader.iter_subareas():
+            if int(geoid.split('/')[1]) > 100:
+                continue
+            if verbose:
+                print(f"Simplifying {geoid}...")
```

Comment:
It would be better to use abseil Python logging (https://abseil.io/docs/python/guides/logging).  

```
from absl import logging

logging.info("Simplifying {geodi}...")
```

The advantage is that the caller can control whether the logging statements are sent to the standard output xor not.  My favorite flag is `--alsologtostderr` so that logging messages are sent to the standard error, not the standard output.  The need for `verbose` disappears.